base/docs/basedocs.jl

```diff
@@ -1797,6 +1797,20 @@ AbstractFloat
     Integer <: Real
 
 Abstract supertype for all integers.
+
+See also [`isinteger`](@ref), [`trunc`](@ref), [`div`](@ref).
+
+# Examples
+```
+julia> 42 isa Integer
+true
+
+julia> 1.0 isa Integer
```

Comment:
Maybe this should mention Int, somehow:
```suggestion
julia> 42 isa Integer  # concrete type Int
true

julia> 1.0 isa Integer  # concrete type Float64
```